### PR TITLE
Change TS/dependencies

### PR DESCRIPTION
**Pull request by ProchaLu**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,43 +23,45 @@
   },
   "devDependencies": {
     "@babel/eslint-parser": "7.17.0",
-    "@next/eslint-plugin-next": "12.1.4",
-    "@types/node": "17.0.23",
-    "@types/react": "17.0.43",
-    "@types/react-dom": "17.0.14",
+    "@next/eslint-plugin-next": "12.1.6",
+    "@types/node": "17.0.31",
+    "@types/react": "^18.0.8",
```
Comment (karlhorky) [inline package.json:28]:
```suggestion
    "@types/react": "18.0.8",
```

File: package.json [hunk 2]
```diff
@@ -23,43 +23,45 @@
...
+    "@typescript-eslint/parser": "5.22.0",
+    "eslint-plugin-import": "2.26.0",
     "eslint-plugin-react": "7.29.4",
-    "eslint-plugin-jest": "26.1.3",
-    "eslint-plugin-react-hooks": "4.4.0",
-    "@typescript-eslint/utils": "5.17.0"
+    "eslint-plugin-jest": "26.1.5",
+    "eslint-plugin-react-hooks": "4.5.0",
+    "@typescript-eslint/utils": "5.22.0",
+    "@types/react": "^18.0.8",
```
Comment (karlhorky) [inline package.json:64]:
```suggestion
    "@types/react": "18.0.8",
```

File: .gitignore [hunk 3]
```diff
@@ -36,6 +36,3 @@ yarn-error.log*
 
 .eslintcache
 *.tsbuildinfo
-
-*.tsbuildinfo
-*.tsbuildinfo
```
Comment (Josehower) [inline .gitignore:41]: maybe delete this extra change here. Is not necesary

**Review by Josehower — CHANGES_REQUESTED** [review]
A small change and i think we are ready

File: .gitignore (hunk 3 above)
Comment (ProchaLu) [reply .gitignore]: Reverted it

**Pull request merged**